internal/jasmine_node_test/jasmine_runner.js

```diff
@@ -63,6 +63,9 @@ function main(args) {
   process.argv.splice(2, 1)[0];
 
   const jrunner = new JasmineRunner({jasmineCore: jasmineCore});
+  if (args.length == 2) {
+    jrunner.loadConfigFile(args[1])
```

Comment:
what happens when the user sets config options that conflict with how we configure jasmine below? if you try to add your own specs, for example? would be good to understand how the rule will behave under all the possible config options, and for those that we override, how will the user know?